packages/build/package.json

```diff
@@ -39,7 +39,6 @@
     "@rspack/core": ">=1.0.5 <2.0.0",
     "@rsbuild/core": ">=1.0.5 <2.0.0",
     "@angular/build": ">=18.0.0 <20.0.0",
-    "sass": "^1.79.3",
```

Comment:
This is incorrect: https://github.com/Coly010/ng-rspack-build/blob/b677061964be9cde1d053581f7dc47c6aff555e0/packages/build/src/lib/rspack/plugins/angular.ts#L8

It is used and should remain as a dep

Reply:
reverted